tests/annotation_regression_examples/script_annotations_artifacts_artifactory_old.py

```diff
@@ -4,7 +4,6 @@
 from hera.workflows.steps import Steps
 
 global_config.experimental_features["script_annotations"] = True
-global_config.experimental_features["script_runner"] = True
```

Comment:
Think it makes sense to turn off the script_annotations feature in the `_old` files too

Reply:
Yes, those are removed now